src/dev-core/src/app/tag/tag.demo.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component } from '@angular/core';
+import '@clr/core/badge';
+import '@clr/core/tag';
+
+@Component({
+  selector: 'app-badge-demo',
+  templateUrl: './tag.demo.html',
+})
+export class TagDemoComponent {
+  open = false;
```

Comment:
What does the `open` property do?

Reply:
What is it good for? Absolutely nothing.

Removing. :)